src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -79,6 +79,8 @@ private static void WriteData(JsonWriter jsonWriter, NuGetIndexSearcher searcher
                 WriteDocumentValue(jsonWriter, "licenseUrl", document, "LicenseUrl");
                 WriteDocumentValue(jsonWriter, "projectUrl", document, "ProjectUrl");
                 WriteDocumentValueAsArray(jsonWriter, "tags", document, "Tags");
+                // WriteDocumentValueAsArray(jsonWriter, "packageTypes", document, "PackageTypes");
```

Comment:
This was an older implementation that didn't save the packageTypes as an object, but as a simple array. Removed.

---
In reply to: [75550543](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75550543) [](ancestors = 75550543)